### ISSUE TITLE
Invisible point(s) in path in API Rerefrence

### ISSUE THREAD

**Issue by Oupsla** [sencrop/sencrop-developer-platform#33]
In reference page, paths of endpoints contain invisible points and sometimes they have kept when copying to clipboard.

This can lead to useless time wasting.

### Screenshots 
![image](https://user-images.githubusercontent.com/15279620/62621728-ed1f8700-b91c-11e9-8e08-4f4c38deaa2a.png)

![image](https://user-images.githubusercontent.com/15279620/62621736-f3adfe80-b91c-11e9-821a-f95d454b2ae6.png)

**Comment by nfroidure** [comment]
Yep, I think they did this to fix layout issues (probably to have line breaking on mobile).

Maybe that there is a CSS way to have line breaking and keep paths pasteable. Otherwise, maybe that a "copy in the clipboard" button would impeach such inconvenance.
